act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 </div>
 
 
-Input Rules List (Composite rules only) {#input-rules-list}
-==========================================================
+### Input Rules (Composite rules only) ### {#input-rules}
 
-A [=composite rule=] uses [=outcomes=] from [=atomic rules=] or other composite rules and applies a logic to them so that a single outcome can be determined for each [=test target=]. The identifiers of all rules used in the [expectations](#expectations-composite) MUST be listed in the composite rule. The rules list describes the input for composite rules, similar to how [aspects under test](#input-aspects) describe the input for atomic rules.
+A [=composite rule=] uses [=outcomes=] from [=atomic rules=] and applies logic to them so that a single [=outcome=] can be determined for each [=test target=]. The [identifier](#rule-identifier) and [=descriptive title=] of all the atomic rules used in the [expectations](#expectations-composite) MUST be listed in the composite rule. The input rules describe the input for composite rules, similar to how [input aspects](#input-aspects) describe the input for atomic rules.
 
 
 Applicability {#applicability}
-==============================
+------------------------------
 
-The applicability describes what (parts of) the [=test subject=] are tested.
+The applicability describes what parts of the [=test subject=] are tested.
 
 
-Applicability for Atomic Rules {#applicability-atomic}
-------------------------------------------------------
+### Applicability for Atomic Rules ### {#applicability-atomic}
 
-The applicability section is a required part of an [=atomic rule=]. It MUST contain a precise description of the parts of the [=test subject=] to which the rule applies. For example, specific nodes in the DOM [[DOM]] tree, or tags that are incorrectly closed in an HTML [[HTML]] document. These are known as the [=test targets=]. The applicability MUST only use information provided through [test aspects](#input-aspects) of the same rule. No other information can be used in the applicability.
+The applicability section is a required part of an [=atomic rule=]. It MUST contain a precise description of the parts of the [=test subject=] to which the rule applies. For example, specific nodes in the DOM [[DOM]] tree, or tags that are incorrectly closed in an HTML [[HTML]] document. These are known as the [=test targets=]. The applicability MUST only use information made available through the listed [input aspects](#input-aspects) in the rule. No other information can be used in the applicability.
 
 Applicability MUST be described objectively, unambiguously and in plain language.
```

Comment:
**Editorial**: Combine this sentence with the paragraph after it. It's about the same topic.